test1/MatrixMemory/MatrixForm.cs

```diff
@@ -0,0 +1,84 @@
+namespace MatrixMemory;
+
+using System.Collections.Generic;
+
+public partial class MatrixForm : Form
+{
+    public MatrixForm()
+    {
+        InitializeComponent();
+
+        game.MakeStart();
+        foreach (var button in buttons)
+        {
+            if (game.MarkedSquares.Contains(button.TabIndex))
+            {
+                button.Text = "X";
+            }
+        }
+
+        var timer = new System.Windows.Forms.Timer();
+        timer.Interval = 5000;
+        timer.Start();
+        DisableAllButtons();
+        timer.Tick += OnTimerTick!;
+    }
+
+    private void OnTimerTick(object sender, EventArgs e)
+    {
+        var timer = (System.Windows.Forms.Timer)sender;
+        timer.Stop();
+
+        foreach (var button in buttons)
+        {
+            if (button.Text == "X")
+            {
+                button.Text = "";
+            }
+        }
+        EnableAllButtons();
+        label1.Text = "Mark all previously marked squares!";
```

Comment:
```suggestion
        currentDescription.Text = "Mark all previously marked squares!";
```
Так себе имя для метки